sql/updates/world/3.3.5/2017_xx_xx_xx_world_335.sql

```diff
@@ -0,0 +1,48 @@
+DROP TABLE IF EXISTS `points_of_interest_locale`;
+CREATE TABLE IF NOT EXISTS `points_of_interest_locale` (
+  `ID` MEDIUMINT(8) UNSIGNED NOT NULL DEFAULT '0',
+  `locale` VARCHAR(4) NOT NULL,
+  `Name` TEXT,
+  `VerifiedBuild` SMALLINT(5) DEFAULT '0'
+) ENGINE=MYISAM DEFAULT CHARSET=utf8;
+
+ALTER TABLE `points_of_interest_locale`
+ADD PRIMARY KEY (`ID`,`locale`);
+
+-- koKR
+INSERT INTO `points_of_interest_locale` (`ID`, `locale`, `Name`)
+	(SELECT `entry`, "koKR", `icon_name_loc1` FROM `locales_points_of_interest`);
```

Comment:
all of these selects could use a `WHERE LENGTH(icon_name_locX) > 0` to avoid inserting empty data if for example only one locale is filled